ref/general/microshed.adoc

```diff
@@ -0,0 +1,80 @@
+:page-layout: general-reference
+:page-type: general
+:page-description: Microservice architecture is a popular approach for building cloud-native applications in which each capability is developed as an independent service. It enables small, autonomous teams to develop, deploy, and scale their respective services independently.
+:page-categories: MicroProfile
+:seo-title: MicroProfile simplifies developing cloud-native Java microservices
+:seo-description: Microservice architecture is a popular approach for building cloud-native applications in which each capability is developed as an independent service. It enables small, autonomous teams to develop, deploy, and scale their respective services independently.
+= Testing in a container
+
+MicroShed testing helps you to write integration tests using testcontainers for Java microservice applications. With microShed testing you can test your Open Liberty application from outside the container so you are testing the exact same image that runs in production.
+
+Integration testing looks as microservices as a group and tests them as a whole unit. Intergration testing offers significant advantages over other types of testing like...
+
+== Why integration testing
+
+In integration testing, you can put together individual pre-tested modules and test them as a group. Integration testing follows unit testing. The modules used are previously unit tested, but now they are now tested as a whole to evaluate compliance and functionality.
+
+
+== What challenges does Microshed resolve
+
+An application might run differently in production as compared to test/dev environment. This can cuse a test to fail in production though it passed in  dev/test. You can address this issue by testing your application in the same container that you’ll use in production.
+
+Running tests in development mode is convenient for local development, but it can be tedious to test against a running Open Liberty server in non-development scenarios such as CI/CD pipelines. For this reason, MicroShed Testing can start and stop the application runtime before and after the tests are run. This process is primarily accomplished by using Docker and Testcontainers.
+
+
+
+
+== How it works (with code snippets)
+
+
+
+You can use MicroShed Testing to develop integration tests for your Open Liberty application. You test your application from outside its container so that the tests run against the same image that you use in production.
+
+To write a test with MircoShed testing, first add microshed-testing-testcontainers and junit-jupiter artifact IDs as test-scoped dependencies, as shown in the following example:
```

Comment:
Suggestion for this sentence:
To write a test with MircoShed testing, first add `microshed-testing-testcontainers` and `junit-jupiter` artifact IDs as test-scoped dependencies, as shown in the following example:

Reply:
The suggestion is the same as the content in the topic. Can you please point to the chnages you are recommending.